source/Calamari.Common/Features/Scripting/DotnetScript/DotnetScriptBootstrapper.cs

```diff
@@ -47,7 +47,7 @@ static DotnetScriptBootstrapper()
             foreach (var executableName in executableNames)
             {
                 var (_, commandOutput) = ExecuteCommandAndReturnOutput(commandLineRunner,
-                                                                       environmentVars,
+                                                                       environmentVars ?? new Dictionary<string, string>(),
```

Comment:
`CommandLineInvocation.EnvironmentVars` is of type ` Dictionary<string, string>?`  , so you can make `ExecuteCommandAndReturnOutput` take in ` Dictionary<string, string>?`  too. No need to create a new dict here